dockstore-integration-testing/src/test/java/io/dockstore/client/cli/WorkflowIT.java

```diff
@@ -693,6 +693,19 @@ public Boolean isPublish() {
         assertTrue("could get mta as part of list", toolV2s.size() > 0 && toolV2s.stream().anyMatch(tool -> Objects
             .equals(tool.getId(), mtaWorkflowID)));
         assertNotNull("could get mta as a specific tool", toolV2);
+
+        // Check that a workflow from my namespace is present
+        assertTrue("Should have at least one repo from DockstoreTestUser2.", workflows.stream().anyMatch((Workflow workflow) -> workflow.getOrganization().equalsIgnoreCase("DockstoreTestUser2")));
```

Comment:
It would be good to have some tests that the columns are what we expect with the change
(i.e. the organization column is what we expect, the name is correct, and both are not concatenated due to the change from name to full name below) 
This might exist already, but let's double check

Reply:
I don't think we have any existing organisation tests, I'll extend these tests I added to check these.

Reply:
Also in the code, the keys are actually never directly used by the code. Technically you could just iterate over the  github.getMyself().listRepositories(pageSize, GHMyself.RepositoryListFilter.ALL), however I was hesitant to change a lot of code.